### ISSUE TITLE
Vim mode is no more working on firefox

### ISSUE THREAD

**Issue by JUNGHUNKIM-7** [dart-lang/dart-pad#2412]
- Problem
Whenever i escape editing from insert mode to normal mode, that doesn't work properly, 
More specifically, escape key isn't working.
i tested another online pad(ts, rust) & browser(edge). but it seems the problem of dartpad. 
originally, i was very satisfied and enjoyed coding at dartpad with vim.

Browser: Firefox
Version: 106.0.5 (x64)
Operating system: windows10

**Comment by parlough** [comment]
Thanks for opening an issue. I see what caused this and am working on a fix.

**Comment by johnpryan** [comment]
This also happens in Chrome for me sometimes - as a workaround you should be able to press `ctrl+c` to get the editor in the right state again.
